doc/contributing/releases.md

```diff
@@ -448,6 +448,23 @@ testing is required. Use this opportunity particularly to test macOS and Windows
 installers if there are any concerns. Click through to the individual workers
 for a run to find the artifacts.
 
+Release build list
+
+| name                                       | OS            |
+| ------------------------------------------ | ------------- |
+| rhel8-release-sources                      | ?             |
```

Comment:
I think would be great to clarify which operating system are each binary even if the name is clear enough

Reply:
While useful, this might be difficult to maintain as it can be different for each major version

Reply:
```suggestion
| rhel8-release-sources                      | Linux x64     |
```